router/throttler/throttler.go

```diff
@@ -0,0 +1,166 @@
+package throttler
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/EagleChen/restrictor"
+	"github.com/rudderlabs/rudder-server/config"
+	"github.com/rudderlabs/rudder-server/utils/logger"
+)
+
+var (
+	parameters map[string]map[string]interface{}
+)
+
+//Throttler is an interface for throttling functions
+type Throttler interface {
+	LimitReached(key string) bool
+	Enabled() bool
+}
+
+//HandleT is a Handle for event limiter
+type HandleT struct {
+	UserEventOrderingRequired bool
+	permit                    bool
```

Comment:
Is this different from the general `enabled` we regularly use? Seems like enabled is a better word than permit.